### ISSUE TITLE
Search in store not working

### ISSUE THREAD

**Issue by royto** [custom-components/hacs#43]
## Version of HACS
0.4.3

## Describe the bug
Search not working in store

## Debug log
![image](https://user-images.githubusercontent.com/6990995/58767046-5209da00-8586-11e9-8a32-1e40b3ee11c3.png)

error in console:
```text
Uncaught TypeError: Search is not a function
    at HTMLInputElement.onkeyup
```

**Comment by ludeeus** [comment]
What type of browser and what version is this?

**Comment by vMeph** [comment]
i can confirm search not working, im using chrome.

**Comment by ludeeus** [comment]
I have only tested with chrome so I know that works.
Try clearing the cache or incognito mode.

**Comment by vMeph** [comment]
@ludeeus thats exactly the problem, now its working, but seems that for some reason the new version removed hacs.css and the hacs.js

**Comment by ludeeus** [comment]
Found the issue, it's downloading to the base of `/custom_components/` working on a fix now.
Next update will also have that issue, but that should be the last version with that issue. :)

**Comment by ludeeus** [comment]
This has been corrected in [`0.4.4`](https://github.com/custom-components/hacs/releases/tag/0.4.4)
Not that when downloading 0.4.4 you will still have that issue. but the next one should now be OK :)
